packages/channels/src/base/meta.ts

```diff
@@ -3,7 +3,8 @@ import Joi from 'joi'
 export interface ChannelMeta {
   id: string
   name: string
+  version: string
   initiable: boolean
   lazy: boolean
-  schema: Joi.ObjectSchema
+  schema: { [field: string]: Joi.Schema }
 }
```

Comment:
Channels now have a version field. Notice they also provider the schema as an object now instead of a complete Joi.Schema. This is to allow us to add the version and enabled field to it.